src/files/files-list/FilesList.js

```diff
@@ -350,7 +351,9 @@ const dropCollect = (connect, monitor) => ({
   canDrop: monitor.canDrop()
 })
 
+export const StoryFilesList = DropTarget(NativeTypes.FILE, dropTarget, dropCollect)(translate('files')(FilesList))
+
 export default connect(
   'selectNavbarWidth',
-  DropTarget(NativeTypes.FILE, dropTarget, dropCollect)(translate('files')(FileList))
+  DropTarget(NativeTypes.FILE, dropTarget, dropCollect)(translate('files')(FilesList))
```

Comment:
Let's rename `StoryFilesList` to be `FilesListWithDropTarget` and reuse it in the default export here.